aspnetcore/host-and-deploy/proxy-load-balancer.md

```diff
@@ -210,6 +210,36 @@ services.Configure<ForwardedHeadersOptions>(options =>
 });
 ```
 
+### Configuration for an IPv4 address represented as an IPv6 address
+
+If a network or proxy address is an IPv4 address nested in an IPv6 address, supply the address to `KnownNetworks` or `KnownProxies` in the nested IPv6 format.
+
+In the following example, two network addresses that supply forwarded headers are added to the `KnownNetworks` in IPv6 format:
+
+* Long format IPv6 for `10.11.12.0/24`:
+  * `0000:0000:0000:0000:0000:ffff:0a0b:0c00`
+  * Prefix length: 24
```

Comment:
The prefix length also needs to be adjusted for the IPv6 format. 24 -> 120, you've added 96 bits to the address.